string label
geometry_msgs/Point xyz

